packages/components/forms/src/BaseCheckbox/types.ts

```diff
@@ -48,4 +48,5 @@ export interface BaseCheckboxInternalProps
    */
   helpText?: string;
   onChange?: ChangeEventHandler<HTMLInputElement>;
+  density?: 'low' | 'high';
```

Comment:
You should be able to replace this with the Density type.

https://github.com/contentful/forma-36/blob/408cd9aadf665d06f5b06856b6cea7ffa6d4c4e8/packages/components/utils/src/useDensity/useDensity.ts?plain=1#L3